test/common/extensions/filters/http/assertion/assertion_filter_test.cc

```diff
@@ -495,6 +497,30 @@ TEST_F(AssertionFilterTest, ResponseTrailersNoMatch) {
   EXPECT_EQ(Http::FilterTrailersStatus::StopIteration, filter_->encodeTrailers(response_trailers));
 }
 
+TEST(AssertionFilterFactoryTest, Config) {
```

Comment:
Maybe this is an upstream envoy convention to name the test minimally (`Config`), but I think it might be nice to make the name more clearly demonstrate what specifically the test is validating. In our platform tests, names are usually closer to sentences describing the behavior

Reply:
😅 yeah convention for this type of very very basic acceptance test. Let me merge and change in subsequent PR.